src/Logout.tsx

```diff
@@ -23,31 +17,15 @@ const Logout: FC = () => {
   const dispatch = useDispatch()
 
   useEffect(() => {
-    const handleReset = () => {
-      dispatch(reset())
-      dispatch({type: 'USER_LOGGED_OUT'})
```

Comment:
I think this action should be explicitly declared somewhere. We have `reset()` declared (and are importing it from the flag action creator), but aren't importing the `USER_LOGGED_OUT` action, which makes it confusing to trace if you're trying to follow dispatched actions through the reducer logic.